man/man5/zfs-events.5

```diff
@@ -777,7 +784,10 @@ Checksum bad offset ranges.
 \fBbad_ranges_min_gap\fR
 .ad
 .RS 12n
-Checksum allowed minimum gap.
+In order to bound the size of the bad_ranges array, gaps of non-corruption
+less than or equal to bad_ranges_max_gap bytes have been merged with adjacent
```

Comment:
`\fB`, `\fR` around the `bad_ranges_max_gap`.

Reply:
yep, also bad_ranges_max_gap is now bad_ranges_min_gap